src/_DomainDriven/Framework.DomainDriven.Core/_Visitors/ExpressionVisitorAggregator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+
+using Framework.Core;
+
+namespace Framework.DomainDriven;
+
+public class ExpressionVisitorAggregator : IExpressionVisitorContainer
```

Comment:
The interface has name ExpressionVisitor**Container** but the class has name ExpressionVisitor**Aggregator**
Why?